### ISSUE TITLE
add #events/lumi in internal FJR object

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#5433]
when running a CMSSW version which put in the XML FJR the output number of events in each lumi, must propagate this information in the internal FJR (JSON?  pickle?) object which is returned to the server

**Comment by emaszs** [comment]
I ran some tasks with the new CMSSW_9_1_0_pre2 version and a new WMCore version which had the code to parse the new style of FJRs , this is the postjob log for one job [1]. On line starting with 
`Thu, 06 Apr 2017 10:08:01 CEST(+0000):DEBUG:PostJob Output file info for output.root: {u'runs': {u'1': {u'666892': 50, ...` you see the dictionary with lumis as keys and event counts as values. This information is coming directly from the fjr generated by WMCore.

On line starting with 
`Thu, 06 Apr 2017 10:08:01 CEST(+0000):DEBUG:PostJob Uploading output metadata for output.root to ...` is printed the output metadata, which contains only lumisections as before. The data here has not changed because we only iterate over the keys in this loop [2], so the event counts are ignored.

Basically, WMCore took care of the new CMSSW FJR and the change should be transparent for us.

[1] https://pastebin.com/raw/8E1Bsw6e
[2] https://github.com/dmwm/CRABServer/blob/581336be3943a10e015a3f43d1dbed3118d323a8/src/python/TaskWorker/Actions/PostJob.py#L2552

**Comment by emaszs** [comment]
Don't think this needs to stay open either
